.github/workflows/python-package.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.0, 3.11, pypy-3.6, pypy-3.7]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11, pypy-3.7]
```

Comment:
Python 3.1 strikes again! :) (see CI test) Use quotes around Python versions to fix.